tests_v2/test_cli_notification_reopen_no_blank_window.py

```
@@ -0,0 +1,299 @@
+                    "reopen-regression",
+                    "--body",
+                    "background-window",
+                ],
+            )
+            _must(output.startswith("OK"), f"Expected notify OK output, got: {output!r}")
+            _wait_for_notification(c, notify_title, target_surface)
+
+            before_reopen = c.window_snapshot()
+            expected_window_count = _window_count(before_reopen)
+
+            reopen = subprocess.run(
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)